lib/websocket-server.js

```diff
@@ -275,6 +275,16 @@ class WebSocketServer extends EventEmitter {
 
     cb(ws);
   }
+
+  /**
+   * Provide address of server
+   * @public
+   */
+  address () {
+    if (!this._server) throw new Error(`Server was not initialized`);
```

Comment:
Use `this.options.noServer` as we don't wan't to throw the error if `wss.address()` is called after `wss.close()`

```js
if (this.options.noServer) {
  throw new Error('This server is operating in "noServer" mode');
}
```